### Post 225
**Post URL**: /t/tds-official-project1-discrepencies/171141/225
- **ID**: 614740
- **Author**: Aryan Tikam (AryanTikam)
- **Created At**: 2025-04-03T03:32:09.872Z
- **Reply To**: Post 224 (Carlton D'Silva, carlton)
- **Content**:  
  <div class="lightbox-wrapper"><a class="lightbox" href="https://europe1.discourse-cdn.com/flex013/uploads/iitm/original/3X/6/a/6a204d9c51d90a584e4e56d593b24821129e056e.png" data-download-href="/uploads/short-url/f8POUeYUc14WP3S01hUZIrrPUOW.png?dl=1" title="1000004190" rel="noopener nofollow ugc"><img src="https://europe1.discourse-cdn.com/flex013/uploads/iitm/optimized/3X/6/a/6a204d9c51d90a584e4e56d593b24821129e056e_2_519x500.png" alt="1000004190" data-base62-sha1="f8POUeYUc14WP3S01hUZIrrPUOW" width="519" height="500" srcset="https://europe1.discourse-cdn.com/flex013/uploads/iitm/optimized/3X/6/a/6a204d9c51d90a584e4e56d593b24821129e056e_2_519x500.png, https://europe1.discourse-cdn.com/flex013/uploads/iitm/optimized/3X/6/a/6a204d9c51d90a584e4e56d593b24821129e056e_2_778x750.png 1.5x, https://europe1.discourse-cdn.com/flex013/uploads/iitm/optimized/3X/6/a/6a204d9c51d90a584e4e56d593b24821129e056e_2_1038x1000.png 2x" data-dominant-color="131313"><div class="meta"><svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">1000004190</span><span class="informations">1193×1149 136 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg></div></a></div><br>
Here it is with the task, however the error doesn’t seem to be related to the task itself based on the returned message in the JSON. It seems to be something wrong with the OpenAI API key. From the reading I did, it seems that the key was perhaps not set properly during evaluation? Not completely sure but please look into it.
- **Reactions**: None
- **Post Number**: 225

